tfhe/docs/getting_started/quick_start.md

```diff
@@ -64,98 +44,101 @@ fn main() {
-    let ct_1 = client_key.encrypt(true);
-    let ct_2 = client_key.encrypt(false);
-
-    // We use the server public key to execute a boolean circuit:
-    // if ((NOT ct_2) NAND (ct_1 AND ct_2)) then (NOT ct_2) else (ct_1 AND ct_2)
-    let ct_3 = server_key.not(&ct_2);
-    let ct_4 = server_key.and(&ct_1, &ct_2);
-    let ct_5 = server_key.nand(&ct_3, &ct_4);
-    let ct_6 = server_key.mux(&ct_5, &ct_3, &ct_4);
-
-    // We use the client key to decrypt the output of the circuit:
-    let output = client_key.decrypt(&ct_6);
-    assert_eq!(output, true);
+    let config = ConfigBuilder::all_disabled()
+        .enable_default_integers()
+        .build();
+
+    let (client_key, server_key) = generate_keys(config);
 }
 ```
 
-#### shortint example
+The `generate_keys` command returns a client key and a server key.
+
+The `client_key` is meant to stay private and not leave the client whereas the `server_key` can be made public and sent to a server for it to enable FHE computations.
 
-Here is a full example using shortint:
+### 2. Setting the server key.
 
-{% hint style="warning" %}
-Use the `--release` flag to run this example (eg: `cargo run --release`)
-{% endhint %}
+The next step is to call `set_server_key`
+
+This function will **move** the server key to an internal state of the crate and manage the details to give a simpler interface.
 
 ```rust
-use tfhe::shortint::prelude::*;
+use tfhe::{ConfigBuilder, generate_keys, set_server_key};
 
 fn main() {
-    // We generate a set of client/server keys
-    // using parameters with 2 bits of message and 2 bits of carry
-    let (client_key, server_key) = gen_keys(PARAM_MESSAGE_2_CARRY_2_KS_PBS);
+    let config = ConfigBuilder::all_disabled()
+        .enable_default_integers()
+        .build();
+
+    let (client_key, server_key) = generate_keys(config);
 
-    let msg1 = 1;
-    let msg2 = 0;
+    set_server_key(server_key);
+}
+```
 
-    let modulus = client_key.parameters.message_modulus().0;
+### 3. Encrypting data.
 
-    // We use the client key to encrypt two messages:
-    let ct_1 = client_key.encrypt(msg1);
-    let ct_2 = client_key.encrypt(msg2);
+Encrypting data is done via the `encrypt` associated function of the \[FheEncrypt] trait.
 
-    // We use the server public key to execute an integer circuit:
-    let ct_3 = server_key.unchecked_add(&ct_1, &ct_2);
+Types exposed by this crate implement at least one of \[FheEncrypt] or \[FheTryEncrypt] to allow enryption.
 
-    // We use the client key to decrypt the output of the circuit:
-    let output = client_key.decrypt(&ct_3);
-    assert_eq!(output, (msg1 + msg2) % modulus as u64);
-}
-```
+```Rust
+let clear_a = 27u8;
+let clear_b = 128u8;
 
-#### integer example
+let a = FheUint8::encrypt(clear_a, &client_key);
+let b = FheUint8::encrypt(clear_b, &client_key);
+```
 
-{% hint style="warning" %}
-Use the `--release` flag to run this example (eg: `cargo run --release`)
-{% endhint %}
+### 4. Computation and decryption.
```

Comment:
numbers don't match the announced outline at the beginning of the file ? i.e. missing 5 and 6 I believe